docs/SETTINGS.rst

```diff
@@ -15,6 +15,7 @@ Dynamic configuration is stored in the DCS (Distributed Configuration Store) and
 -  **ttl**: the TTL to acquire the leader lock (in seconds). Think of it as the length of time before initiation of the automatic failover process. Default value: 30
 -  **retry\_timeout**: timeout for DCS and PostgreSQL operation retries (in seconds). DCS or network issues shorter than this will not cause Patroni to demote the leader. Default value: 10
 -  **maximum\_lag\_on\_failover**: the maximum bytes a follower may lag to be able to participate in leader election.
+-  **max\_timelines\_history**: maximum timelines history should be kept in DCS. Default value: 0. When set to 0, it keeps all timelines history in dcs.
```

Comment:
I suggest changing the wording to 
'...: maximum number of timeline history items kept in DCS.  Default value: 0. When set to 0, it keeps the full history in DCS.'